SynthCh10Gen/Source_CsvTxt.cpp

```diff
@@ -249,7 +253,11 @@ bool ClSource_CsvTxt::ConvertTime(std::string sTime, double *fDecodedTime)
     suNasaTime.tm_sec   = (int)fSecond;
 
     // Convert to a time_t
+    #ifdef LINUX_BUILD
+    lNasaTime = mktime(&suNasaTime);
```

Comment:
is mktime on linux the same as _mkgmtime on windows?

Reply:
yes

Reply:
i'm worried about time zones.  specifically UTC, I can't immediately tell from the docs whether these functions are the same.

Reply:
Yeah, I think you're right. apparently I didn't research enough before I changed it.